frameworklauncher/src/main/java/com/microsoft/frameworklauncher/common/model/TaskStatus.java

```diff
@@ -41,7 +41,8 @@
   private String containerHost;
   // containerIp is the assigned ipv4 address of the corresponding containerHost
   private String containerIp;
-  private List<ValueRange> containerPorts;
+  // containerPorts is a string contains ports for each port label, such as "httpPort:80,81,82;sshPort:1021,1022,1023;"
+  private String containerPorts;
```

Comment:
Is there a trailing `;` at the end of string or not?

Reply:
yes

---
In reply to: [183324757](https://github.com/Microsoft/pai/pull/495#discussion_r183324757) [](ancestors = 183324757)